src/app.js

```diff
@@ -205,7 +205,7 @@ const coreBuildRoutes = coreBuildPaths.map(path => [
   `${path}:*`, // Tangletrees at top-level, e.g. /a:/a/b
 ]);
 
-app.use([coreBuildRoutes, "/narratives/*"], setSource(req => new CoreSource()));
+app.use([coreBuildRoutes, "/narratives/*"], setSource(_req => new CoreSource()));
```

Comment:
Changes like this make the linting rule seem a bit silly to me. It's plainly obvious to any reader that `req` isn't used without needing the `_` prefix. We're essentially adding the `_` prefix to the name for the computer alone, as a replacement for adding `// eslint-disable-line no-unused-vars` instead.

Reply:
I'll re-add the existing per-line exceptions but keep the rule enabled. Does that sound good? To me, this rule is still useful for catching things such as unused imports and other unintended unused variables.

Reply:
Sure. I guess I really wish we had a better rule. (We could, but would have to write it, it seems.)